### DISCUSSION TITLE
Skeleton, Browser Sessions

### DISCUSSION THREAD

**Discussion by cjmellor** [HassanZahirnia/laravel-package-ocean#122] [Package Suggestions]
Hi,

Can I put forward two more of my own packages to be added?

**Skeleton**
https://github.com/cjmellor/skeleton

**Browser Sessions**
https://github.com/cjmellor/browser-sessions

Thanks!

**Comment by HassanZahirnia** [comment]
Hey Chris, thanks for your suggestions!
Unfortunately the first repo cannot be added because every package on the website need to have a composer/npm package otherwise it won't work.

As for the second one, I'm gonna give it a bit more time as I'm not sure if the community will find it useful. I feel like it needs to be more feature packed, it looks too slim as a package. But that's just my opinion.

**Reply by cjmellor** [reply]
It makes sense regarding: the skeleton

I have some ideas for further features for the other package.

No harm in trying aye?

**Reply by HassanZahirnia** [reply]
Yeah no problem, let me know when you've added a few more features to it. 😊👍

**Reply by cjmellor** [reply]
I added a feature where users can see how long since last activity. 

Also, package was featured on Laravel News and has 100+ stars.

**Reply by HassanZahirnia** [reply]
Thanks for the update Chris😊 I've added the package!
